<launch>

  <node name="third_robot_driver_node" pkg="third_robot_driver" type="third_robot_driver_node" 
		output="screen">
    <param name="imcs01_port" type="string" value="/dev/urbtc0" />
	<param name="pulse_rate" type="double" value="40.0" />
	<param name="geer_rate" type="double" value="33.0" />
	<param name="wheel_diameter_right" type="double" value="0.275" />
	<param name="wheel_diameter_left" type="double" value="0.280" />
	<param name="tred_width" type="double" value="0.559" /> <!-- 0.611 -->
  </node>
  
  <node name="rossrial_python" pkg="rosserial_python" type="serial_node.py"
		output="screen">
	<param name="port" value="/dev/ttyACM0" />
  </node>

</launch>
